test/unittest/tests/block/test_block_other.cpp

```diff
@@ -19,27 +19,23 @@
 static void
 test_block_header_byteswap_copy(WT_BLOCK_HEADER *from, WT_BLOCK_HEADER *to)
 {
-    WT_BLOCK_HEADER prev_from, prev_to;
+    WT_BLOCK_HEADER prev_from;
 
     // Save the original values before any potential byte re-orderings.
     prev_from.disk_size = from->disk_size;
     prev_from.checksum = from->checksum;
-    prev_to.disk_size = to->disk_size;
-    prev_to.checksum = to->checksum;
 
-#ifdef WORDS_BIGENDIAN
     __wt_block_header_byteswap_copy(from, to);
```

Comment:
`__wt_block_header_byteswap_copy` is now outside of `#ifdef WORDS_BIGENDIAN`. 
When we call it in little endian mode it just copies the contents of `from` into `to` with no byte swapping. Previously we weren't testing the workflow.